liquid_node/configuration.py

```diff
@@ -160,6 +160,8 @@ def __init__(self):
         self.tika_count = self.ini.getint('liquid', 'tika_count', fallback=1)
         self.tika_memory_limit = self.ini.getint('liquid', 'tika_memory_limit', fallback=800)
 
+        self.thumbnail_memory_limit = self.ini.getint('liquid', 'thumbnail_memory_limit', fallback=800)
+
```

Comment:
Please add another parameter for the `count` (like we have with Tika, default 1). This is needed to scale on big machines with tens of CPUs, or clusters with many small machines.